pkg/webhook/admission/pod/mutator.go

```diff
@@ -108,3 +107,12 @@ func (mutator *Mutator) mutate(pod *v1.Pod, configMap *v1.ConfigMap) error {
 
 	return nil
 }
+
+func needMutate(pod *v1.Pod) bool {
+	// Skip webhook if pod not managed by kfserving
+	if _, ok := pod.Labels[constants.InferenceServicePodLabelKey]; ok {
```

Comment:
just return ok?

Reply:
in fact, here `ok` is inferenceservice name, which is a string. I think we had better return bool

Reply:
updated